src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-sync-data-info/rgw-sync-data-info.component.html

```diff
@@ -40,24 +43,9 @@
        placement="top"
        popoverClass="rgw-overview-card-popover"
        i18n>Error</a></li>
-  <li class="mt-4 w-100 text-center"
-      *ngIf="zone.syncstatus === 'preparing for full sync'">
-    <b>Full sync progress:</b>
-    <cd-usage-bar *ngIf="zone.fullSync"
-                  [total]="zone.fullSync[1]"
-                  [showMultisiteTooltip]="true"
-                  [used]="zone.fullSync[0]"
-                  [title]="shards"
-                  decimals="2">
-    </cd-usage-bar></li>
-  <li class="mt-4 w-100 text-center"
-      *ngIf="zone.incrementalSync">
-  <b>Sync Progress:</b>
-  <cd-usage-bar *ngIf="zone.incrementalSync"
-                [total]="zone.totalShards"
-                [showMultisiteTooltip]="true"
-                [used]="zone.usedShards"
-                [title]="shards"
-                decimals="2">
-  </cd-usage-bar></li>
+  <li class="mt-4">
+    <b>Last Synced:</b>
+  </li>
```

Comment:
you can use the [bootstrap class](https://getbootstrap.com/docs/5.0/utilities/text/#font-weight-and-italics) `fw-bold` and then remove the `<b>` tag.
```suggestion
  <li class="mt-4 fw-bold">
    Last Synced:
  </li>
```